mslib/static/templates/user/reset_request.html

```diff
@@ -0,0 +1,63 @@
+{% extends "theme.html" %} {% block body %}
+  <div class="container">
+    <style>
+        @media (min-width: 1px) {
+          .container {
+            max-width: 90%;
+          }
+        }
+    </style>
+    <div class="col">
+      <br>
+      <h3>Find your MSS account</h3>
```

Comment:
Recovering a lost MSColab password

If you have forgotten your password, please enter your email address and click "Reset password"
You will receive an email containing a recovery token that you can use to change your password. The email will also contain further explanations.